### DISCUSSION TITLE
Github Feed on Mobile App

### DISCUSSION THREAD

**Discussion by AntonioFuziy** [community/community#14077] [Other Feature Feedback, Questions, & Ideas]
It would be very nice if github mobile app had a feed such as the feed or for you pages from the github website or something in this scenario.

**Comment by benjiroooo** [comment]
This would be nice. Good replacement for Tiktok lol

**Comment by jfullstackdev** [comment]
and there is a new update on the For You feed, you can now filter it, and for those who don't want it can simply disable it by unchecking all filters

![image](https://user-images.githubusercontent.com/47092464/182012117-0b19be9d-e849-462e-909e-d20d28915c1d.png)
